server/src/main/java/org/elasticsearch/rest/action/search/SearchResponseMetrics.java

```diff
@@ -14,6 +14,10 @@
 
 import java.util.Map;
 
+/**
+ * Metrics concerning a set of search responses.
+ * Contains response count, status and timing.
+ */
```

Comment:
I would vote "no" on this comment as it doesn't say anything that you can't tell from the class name and quick look at the instance vars. I generally like adding comments only if they add useful additional context or information.

Reply:
I wanted to explain that this is the container class that aggregates metrics, it wasn't obvious for me from just the name when reading the code. Maybe I need a better description here?

Reply:
At at minimum I would remove the second sentence about what it contains, as that could easily get out of date. It's better to read the code for that sort of thing, I think.